### DISCUSSION TITLE
Followers and following rank token

### DISCUSSION THREAD

**Discussion by khaliddd** [sbertix/Swiftagram#118] [Implemented ideas] [answered]
Hello, 

I want to check if there is any change in followers list since last fetch, to achieve this I am comparing pagination.next of first page

will it work? and will it cover all the scenarios ? 
for example if a user unfollowed from page 1 and one new user followed. will I get different pagination.next since last fetch. 

can you please guide.

**Comment by sbertix** [comment, marked as answer]
Hey @khaliddd,

I suppose when using custom `rank_token` you should be able to compare the behavior of pagination responses. 
As of right now this feature is no longer in **Swiftagram**, but if it's something you indeed require I can totally add it back and release it by the end of the day. 

Let me know.

---

Edit: Added in `4.2.6`.

**Reply by khaliddd** [reply]
@sbertix that will be great, if you can add it.

**Reply by sbertix** [reply]
The feature is now in `4.2.6`. 
I don't have time to make sure it still works as it used to, unfortunately, so you should test it yourself to see if Instagram broke it recently.

**Comment by khaliddd** [comment]
@sbertix how can I get rank_token ?

**Reply by sbertix** [reply]
A constant `rank_token` is automatically applied right now, so theoretically you shouldn't even need to worry about it as it's guaranteed to stay the same for a given `Secret`. 
If you want more control though, you can just use whatever `String` you want. I suggest creating a custom `UUID`, storing it somewhere and relying on it 😊

**Reply by khaliddd** [reply]
okay, thanks
